core/Cargo.toml

```diff
@@ -27,6 +27,9 @@ bs58 = { version = "0.4" , optional = true }
 base64 = { version = "0.22", optional = true }
 serde_json = { version = "1.0", optional = true }
 hex = { version = "0.4" , optional = true }
+hkdf = "0.12"
+sha2 = "0.10"
+hex-literal = "0.4"
```

Comment:
Is this required or just for tests?

Reply:
good point, it was used while developing for tested, it's not used atm. Removed.